modules/virt-defining-storageclass.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-preparing-cdi-scratch-space.adoc
+
+[id="virt-defining-storageclass_{context}"]
+= Defining a storage class
+
+Dynamically provision scratch space for CDI operations by defining a storage class in the `HyperConverged` custom resource (CR).
+
+
+.Prerequisites
+
+* Install the OpenShift client (`oc`).
+
+.Procedure
+
+. Edit the `HyperConverged` CR by running the following command:
++
+[source,terminal]
+----
+$ oc edit hco -n openshift-cnv kubevirt-hyperconverged
+----
+
+. Edit the `spec.scratchSpaceStorageClass` field to match a storage class in the cluster:
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  scratchSpaceStorageClass: "<storage_class>" <1>
+----
+<1> Replace `<storage_class>` with the storage class that you want to use to provision scratch space for CDI operations in your cluster.
+
+. Save and exit your default editor to update the `HyperConverged` CR.
```

Comment:
We may want to mention that when this value is not specified, scratch space will be allocated from the same storage class as the pvc that is being populated.